Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java

```diff
@@ -67,53 +63,55 @@
     protected final static String MOCK_SUBSCRIPTION = "00000000-0000-0000-0000-000000000000";
 
     public static Boolean IS_MOCKED = IsMocked();
-    
+    private static String azureAuthFile = getAuthFile();
+    public static Boolean NetworkStateOn=true;
```

Comment:
This flag is used for simulate network state cases. If the state is false, all request will send to a not existed host.